### PR TITLE
Create Proposal for Million Lumen Homepage

### PR DESCRIPTION
**Pull request by apaldiwal**

### CODE REVIEW

**Comment by rahimklaber** [comment]
I think more could be said about what is expected. For example, necessary knowledge of Stellar Turrets is mentioned, which implies that TSS should be used, but this isn't explicitly mentioned.

**Comment by apaldiwal** [comment]
> I think more could be said about what is expected. For example, necessary knowledge of Stellar Turrets is mentioned, which implies that TSS should be used, but this isn't explicitly mentioned.

Yeah, I have removed it from the description.

File: bounties/level-2/million-lumen-homepage.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+The developer should have knowledge of Web Technologies and Stellar SDK.
+
+### What are the deliverables?
+
+*Description* <br>
+  * The developer should provide a link to a repository implementing the new functionality.
+
+*Judging Criteria and Metrics* <br>
+  * MVP - Does the functionality match with what's stated in the proposal? And how similar it is to the original concept?
```
Comment (tyvdh) [inline bounties/level-2/million-lumen-homepage.md:25]: > stated in the proposal

What proposal? I assume maybe the wikipedia page? The proposal should probably be directly linked to here.

File: bounties/level-2/million-lumen-homepage.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  * UX - Is the UI intuitive and user friendly? Does it adhere to design standards for UI elements and interactions?
+  
+*Submission Procedure* <br>
+  * Submit a link to your repository as a comment on the specific issue thread for this bounty.
```
Comment (tyvdh) [inline bounties/level-2/million-lumen-homepage.md:29]: We've switched from comments on an Issue to a PR on the bounty doc itself. (see https://github.com/tyvdh/stellar-quest-bounties/blob/main/bounties/level-2/stellar-lab-darkmode.md#judging-criteria-and-metrics)

I do think we're going to want to open an Issue as well but for each person on a specific bounty for tracking progress / payout etc. It's important both to be able to track completed submissions as well as intentions for submissions so would be hunters know who else if anyone is already working on this bounty.

Would love to hear more thoughts around the actual flow for bounty hunting.

File: bounties/level-2/million-lumen-homepage.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+
+# Million Lumen Homepage
+
+| Hunter | Agent
+| :- | :-
+| 800 XLM | 2 Credits
```
Comment (tyvdh) [inline bounties/level-2/million-lumen-homepage.md:6]: Let's make this 2000 XLM and 4 Credits

**Review by tyvdh — COMMENTED** [review]
I love this idea. There's lots of room here for innovation and mixing and matching Stellar with maybe IPFS or Arweave for data storage. Or maybe the page is constructed out of memo identifiers of payments which any site could construct. idk. There are some interesting opportunities. Just a few tweaks and maybe take the opportunity to improve our flow and I think this one is ready.

**Comment by apaldiwal** [comment]
> I love this idea. There's lots of room here for innovation and mixing and matching Stellar with maybe IPFS or Arweave for data storage. Or maybe the page is constructed out of memo identifiers of payments which any site could construct. idk. There are some interesting opportunities. Just a few tweaks and maybe take the opportunity to improve our flow and I think this one is ready.

I thought of the IPFS storage option at first but wouldn't it be better to allow the bounty hunters to do their own thing? There are tons of ways one can go about in creating it. The best submission wins the bounty?

**Comment by rahimklaber** [comment]
> I thought of the IPFS storage option at first but wouldn't it be better to allow the bounty hunters to do their own thing? There are tons of ways one can go about in creating it. The best submission wins the bounty?

Maybe you could add it as an suggestion/idea for how the bounty could be done.

File: bounties/level-2/million-lumen-homepage.md (hunk 2 above)
Comment (hanseartic) [reply bounties/level-2/million-lumen-homepage.md]: > https://github.com/tyvdh/stellar-quest-bounties/blob/main/bounties/level-2/stellar-lab-darkmode.md#judging-criteria-and-metrics

https://github.com/tyvdh/stellar-quest-bounties/blob/main/bounties/level-2/%F0%9F%9A%A8-stellar-lab-darkmode.md

**Comment by tyvdh** [comment]
Yeah let's add an ideas header with just a few thoughts around suggesting some interesting potential solutions, tools, concepts etc.

Similar to what I did here:
https://github.com/tyvdh/stellar-quest-bounties/blob/main/bounties/level-2/stellar-quest-badge-checker.md#ideas

**Pull request merged**